de.dlr.sc.virsat.model.edit.test/src/de/dlr/sc/virsat/model/concept/types/roles/BeanDisciplineTest.java

```diff
@@ -46,10 +49,18 @@ public void testGetAndSet() {
 		beanDiscipline.setDiscipline(testDiscipline);
 		assertEquals(testDiscipline, beanDiscipline.getDiscipline());
 		assertEquals(testDiscipline.getUuid().toString(), beanDiscipline.getUuid());
+
+	    // Create an expected list of users
+	    EList<String> expectedUsers = new BasicEList<>();
+		assertEquals(expectedUsers, beanDiscipline.getUsers());
```

Comment:
To address this, i configure Eclipse to display whitespace characters by going to Window > Preferences > General > Editors > Text Editors, and then enabling the "Show whitespace characters" option. This will help ensure consistent formatting throughout the code.